cluster-autoscaler/go.mod

```diff
@@ -198,60 +198,60 @@ replace github.com/digitalocean/godo => github.com/digitalocean/godo v1.27.0
 
 replace github.com/rancher/go-rancher => github.com/rancher/go-rancher v0.1.0
 
-replace k8s.io/api => k8s.io/api v0.26.0
+replace k8s.io/api => k8s.io/api v0.26.1
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.0
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.1
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.26.1-rc.0
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.26.2-rc.0
 
-replace k8s.io/apiserver => k8s.io/apiserver v0.26.0
+replace k8s.io/apiserver => k8s.io/apiserver v0.26.1
 
-replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.0
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.1
 
-replace k8s.io/client-go => k8s.io/client-go v0.26.0
+replace k8s.io/client-go => k8s.io/client-go v0.26.1
 
-replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.26.0
+replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.26.1
 
-replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.26.0
+replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.26.1
 
 replace k8s.io/code-generator => k8s.io/code-generator v0.26.2-rc.0
 
-replace k8s.io/component-base => k8s.io/component-base v0.26.0
+replace k8s.io/component-base => k8s.io/component-base v0.26.1
 
-replace k8s.io/component-helpers => k8s.io/component-helpers v0.26.0
+replace k8s.io/component-helpers => k8s.io/component-helpers v0.26.1
 
-replace k8s.io/controller-manager => k8s.io/controller-manager v0.26.0
+replace k8s.io/controller-manager => k8s.io/controller-manager v0.26.1
 
 replace k8s.io/cri-api => k8s.io/cri-api v0.26.2-rc.0
 
-replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.0
+replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.1
 
-replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.0
+replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.1
 
-replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.0
+replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.1
 
-replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.0
+replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.1
 
-replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.0
+replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.1
 
-replace k8s.io/kubectl => k8s.io/kubectl v0.26.0
+replace k8s.io/kubectl => k8s.io/kubectl v0.26.1
 
-replace k8s.io/kubelet => k8s.io/kubelet v0.26.0
+replace k8s.io/kubelet => k8s.io/kubelet v0.26.1
 
-replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.26.0
+replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.26.1
 
```

Comment:
the solution itself depends upon revendor against 1.26.3
Kindly fix the IT accordingly

Reply:
Yes, re-vendored against 1.26.3 and fixed IT. `adjustNodeGroups` and `scaleDown` were throwing issues due to autoscaler moving coredns to test nodes. This was resolved and re-tested twice.